src/serviceWorker/document.test.ts

```diff
@@ -106,15 +106,15 @@ describe('document', () => {
       let fetched: Response
       const FETCHED_ETAGS = 'fetched'
 
+      const assertFetch = () => {
```

Comment:
This should be more verbose; and it is an expectation, not an assertion: `expectFetchToHaveBeenCalledWithRequestUrl`

Reply:
Always to the rescue with proper name!